docs/ru/operations/table_engines/jdbc.md

```diff
@@ -0,0 +1,82 @@
+# JDBC
+
+Allows ClickHouse to connect to external databases via [JDBC](https://en.wikipedia.org/wiki/Java_Database_Connectivity).
+
+To implement JDBC connection, ClickHouse uses the separate program [clickhouse-jdbc-bridge](https://github.com/alex-krash/clickhouse-jdbc-bridge). You should run it as a daemon.
```

Comment:
needs some additional installation instructions as it's not bundled by default + mention that it's a third-party program